graviti/dataframe/column/series.py

```diff
@@ -11,15 +11,27 @@
 
 import pyarrow as pa
 
+import graviti.portex as pt
 from graviti.dataframe.column.indexing import ColumnSeriesILocIndexer, ColumnSeriesLocIndexer
-from graviti.portex import PortexType
+from graviti.dataframe.container import Container, ContainerRegister
 from graviti.utility import MAX_REPR_ROWS
 from graviti.utility.paging import PagingList
 
 _T = TypeVar("_T", bound="Series")
 
 
-class Series:
+@ContainerRegister(
```

Comment:
Only builtin types?

Reply:
I will implement another register for extension type